### PR TITLE
feat: add chat latency and api: generate prompt starters

### PR DESCRIPTION
**Pull request by Abirdcfly**
#### What type of PR is this?
/kind feature

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: apiserver/pkg/chat/chat_server.go [hunk 1]
```diff
@@ -206,4 +204,184 @@ func (cs *ChatServer) GetMessageReferences(ctx context.Context, req MessageReqBo
...
+			res := strings.Split(result, "\n")
+			for _, r := range res {
+				promptStarters = append(promptStarters, strings.TrimSpace(r))
+			}
+		}
+	}
+	return promptStarters, nil
+}
+
+const PromptForGeneratePromptStarters = `You are the friendly and curious questioner, please ask {{.limit}} questions based on the name and description of this app below.
```
Comment (Abirdcfly) [inline apiserver/pkg/chat/chat_server.go:355]: any idea about this prompt?

**Comment by bjwswang** [comment]
@Abirdcfly  can you rebase the the latest branch and re-generate the swagger docs

**Comment by Abirdcfly** [comment]
> @Abirdcfly can you rebase the the latest branch and re-generate the swagger docs

Done

File: apiserver/pkg/chat/chat_server.go (hunk 1 above)
Comment (nkwangleiGIT) [reply apiserver/pkg/chat/chat_server.go:355]: looks good enough, here is what AI suggests.
```
You are the friendly and curious questioner, your task is to ask 3 thoughtful questions based on the name and description of this app.

Name of the application: 英语老师
Description: 我将跟你交流英语学习中的各类问题，包括词汇、语法、文化等。

Requires language consistent with the name and description of the application, no restating of my words, questions only, one question per line, no subheadings.

The question you asked is:
```

**Pull request merged**